low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -0,0 +1,360 @@
+            returncode = os.system(sspl_rpm_cmd)
+        else:
+            print("INFO: Installing SSPL RPMS using yum repos...")
+            sspl_cmd = "yum install -y cortx-sspl.noarch"
+            sspl_test_cmd = "yum install -y cortx-sspl-test"
+            returncode = os.system(sspl_cmd)
+            if returncode != 0:
+                raise DeployError(returncode, "Failed to install 'cortx-sspl'")
+            returncode = os.system(sspl_test_cmd)
+            if returncode != 0:
+                raise DeployError(returncode, "Failed to install 'cortx-sspl-test'")
+
+        print("INFO: INSTALLING required python3 site-packages..")
+        cmd = "python3 -m pip install -r /opt/seagate/cortx/sspl/low-level/requirements.txt"
+        _ = os.system(cmd)
+
+        print("INFO: Done installing SSPL RPMS.")
+
+    def update_config(self):
+        """Update sspl and sample global config values."""
+        from cortx.utils.conf_store import Conf
+        SSPL_BASE_DIR = "/opt/seagate/cortx/sspl"
+
+        self.sspl_config = "%s/conf/sspl.conf.%s.yaml" % (SSPL_BASE_DIR, self.PRODUCT_VERSION)
+        self.sample_global_config = "%s/conf/sample_global_cortx_config.yaml" % (SSPL_BASE_DIR)
+        if not os.path.exists(self.sspl_config):
+            raise DeployError(1, "sspl config file not found.")
+        if not os.path.exists(self.sample_global_config):
+            raise DeployError(1, "Sample global config file not found.")
+
+        print("INFO: Updating config values in config files..\n" + \
+            "SSPL_CONFIG: %s \nGLOBAL_CONFIG: %s" % (self.sspl_config, self.sample_global_config))
+        Conf.load("sspl", "yaml://%s" % (self.sspl_config))
+        Conf.load("global_config", "yaml://%s" % (self.sample_global_config))
+
+        Conf.set("sspl", "RABBITMQINGRESSPROCESSOR>username", self.RMQ_USER)
+        Conf.set("sspl", "RABBITMQEGRESSPROCESSOR>username", self.RMQ_USER)
+        if self.RMQ_PASSWD:
+            Conf.set("sspl", "RABBITMQINGRESSPROCESSOR>password", self.RMQ_PASSWD)
+            Conf.set("sspl", "RABBITMQEGRESSPROCESSOR>password", self.RMQ_PASSWD)
+            Conf.set("sspl", "LOGGINGPROCESSOR>password", self.RMQ_PASSWD)
+        Conf.set("global_config", "storage>enclosure_1>controller>primary>ip", self.CNTRLR_A)
+        Conf.set("global_config", "storage>enclosure_1>controller>primary>port", self.CNTRLR_A_PORT)
+        Conf.set("global_config", "storage>enclosure_1>controller>secondary>ip", self.CNTRLR_B)
+        Conf.set("global_config", "storage>enclosure_1>controller>secondary>port", self.CNTRLR_B_PORT)
+        Conf.set("global_config", "storage>enclosure_1>controller>user", self.CNTRLR_USER)
+        Conf.set("global_config", "storage>enclosure_1>controller>secret", self.CNTRLR_PASSWD)
+        Conf.set("global_config", "storage>enclosure_1>type", self.STORAGE_TYPE)
+        Conf.set("global_config", "cluser>srvnode_1>node_type", self.SERVER_TYPE)
```

Comment:
Instead of using "srvnode_1"  key directly, you can use the machine-id to fetch node_name(srvnode_1,srvnode_2 etc).

eg: machine_id=cat /etc/machine-id
srvnode=Conf.get("global_config", f"cluster>{machine_id}")

Reply:
Thanks, it helps.

Reply:
This we read from sample_global_config.yaml for single node provisioning.
Dynamically feeding machine_id and mapping a specific name to that machine_id is again a hardcoded thing at this time.
So we used to keep srvnode_1.

Reply:
@satish-seagate , As Sandeep & Shriya mentioned, sspl-ll service uses the machine id from config file, we need to update it.

Reply:
Done